report/render_report.R

```diff
@@ -1,67 +1,41 @@
 library(tidyverse)
 library(dotenv)
-library(REDCapR)
-library(lubridate)
-library(rmarkdown)
 library(sendmailR)
 library(redcapcustodian)
 library(argparse)
 
 init_etl("render_report")
 
 parser <- ArgumentParser()
-parser$add_argument("script_name", nargs=1, help="Script to be run")
+parser$add_argument("script_path", nargs=1, help="The full path of the script to be run")
+
 if (!interactive()) {
   args <- parser$parse_args()
-  script_name <- args$script_name
-  if(!fs::file_exists(script_name)) {
-    stop(sprintf("Specified file, %s, does not exist", script_name))
+  script_path <- args$script_path
+  if(!fs::file_exists(script_path)) {
+    stop(sprintf("Specified file, %s, does not exist", script_path))
   }
```

Comment:
Could you move this block and lines 20-22 outside the if-else block? They look like exact matches.